web/pdf_sidebar.js

```diff
@@ -104,7 +106,7 @@ var PDFSidebar = (function PDFSidebarClosure() {
     reset: function PDFSidebar_reset() {
       this.isInitialViewSet = false;
 
-      this.close();
+      this._hideUINotification(null);
```

Comment:
> I'm fine with the removal of `this.close()` as long as no negative side-effects occur, such as an open sidebar where the view is not reset when a document is opened that has no outline/attachments anymore.

On reset, were already switching to the Thumbnail View (this logic is *not* changed in the patch); see https://github.com/mozilla/pdf.js/blob/master/web/pdf_sidebar.js#L108.
Once the Outline/Attachments have been loaded, we will also switch to the Thumbnail View if the other views are empty; see https://github.com/mozilla/pdf.js/blob/master/web/pdf_sidebar.js#L346-L348 and https://github.com/mozilla/pdf.js/blob/master/web/pdf_sidebar.js#L356-L358.